API.md

```diff
@@ -99,7 +99,7 @@ Bearer token when querying the HTTP API endpoints.
 
 In order for the access token to be accepted, two conditions must be met:
 1. The OpenID Provider recognizes the token as valid and active.
-2. The token must be able to be mapped to a user, using either `irods_user_claim` or `user_attribute_mapping`. 
+2. The token must be able to be mapped to a user using a valid user mapping plugin.
```

Comment:
perhaps?   please tweak phrasing if there is a better way...

```suggestion
2. The token must be able to be mapped to a local zone user using a valid user mapping plugin.
```

Reply:
Does 'local zone' appear anywhere else in the docs? Asking because I haven't been watching restrictions like this closely.

Would there ever be a case where we match outside the local zone?

Reply:
Might need to think through a bit more, OAuth might behave differently that native auth...

Reply:
@korydraughn Did you have any thoughts on this?

Reply:
Looking at the implementation of `get_connection()` in common.cpp, @trel's suggestion is more explicit and correct. I feel the use of `local zone` is mostly covered by the very first statement in the README.

With some tweaks, I believe the HTTP API could allow any user (local and remote) to access the zone. However, supporting remote users isn't a goal of this PR.

As of right now, users will always resolve to `<username>#<zone_from_config_file>`. The `<zone_from_config_file>` is controlled by the HTTP API.

The iRODS server itself always requires users from a different zone to enter the local zone through their respective zones.

Does that answer your question?

Reply:
Yep, just wanted to make sure it all matched up.

Reply:
Done.